mongodbatlas/resource_mongodbatlas_serverless_instance.go

```diff
@@ -100,6 +100,14 @@ func returnServerlessInstanceSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Computed: true,
 		},
+		"connection_strings_private_srv": {
+			Type:     schema.TypeList,
```

Comment:
[q] isn't this a string?

Reply:
oh this one is `privateEndpoint`, can we add `privateEndpoint` to the name to make it more clear?

Reply:
Renamed to connection_strings_private_endpoint_srv